crates/sui-types/src/unit_tests/intent_tests.rs

```diff
@@ -61,12 +61,14 @@ fn test_authority_signature_intent() {
     let recipient = dbg_addr(2);
     let object_id = ObjectID::random();
     let object = Object::immutable_with_id_for_testing(object_id);
-    let data = TransactionData::new_transfer_sui_with_dummy_gas_price(
+    let gas_price = 1000;
```

Comment:
Should we just make this a constant, `TEST_GAS_PRICE`, in `messages` that is set to 1000 to be used everyhwere

Reply:
the value in this test is dummy, it can be any value (not going through the network). We do have a constant value which is 100 in this commit